lib/cisco_node_utils/cmd_ref/bridge_domain.yaml

```diff
@@ -0,0 +1,52 @@
+# bridge_domain
+# bridge_domain feature is available only on n7k
+---
+_exclude: [N3k, N5k, N6k, N9k, ios_xr]
+
+all_bds:
+  multiple: true
+  get_command: "show running-config bridge-domain"
+  get_value: '/^bridge-domain (\S+)/'
+
+create:
+  set_value: "bridge-domain <crbd>"
+
+destroy:
+  set_value: "no bridge-domain <delbd> ; system bridge-domain remove <rembd>"
+
+# Configured fabric-control Bridge-Domain/VLAN    :  100
+fabric_control:
+  get_command: "show bridge-domain summary"
```

Comment:
It looks like you changed some of the properties to use running-config, but not all?  Is it feasible to use the running config for all properties?